manubot/cite/citekey.py

```diff
@@ -98,7 +101,8 @@ def _standardize(self):
             self._standard_prefix = None
             self._standard_accession = None
             self._standard_id = self.dealiased_id
-            return
+            return None
+        # currently this will lowercase unhandled prefixes, do we want this?
```

Comment:
in https://github.com/manubot/manubot/pull/225/commits/0b156cbb5b6f0da785af5fc2671c0cc84992699f, i switched it so unhandled citekeys use dealiased_id for standard_id rather than `prefix_lower:accession`.

For example, `@This-IS-not-a:p:r:e:f:i:x` should remain untouched. It's best if we don't change citekeys that are unhandled, since there is no justification to.

> What are the effects of lowercasing the unhandled prefixes?

Not much because it would happen for both manual references and citekeys. But it could add extra confusion.